aiida/orm/implementation/storage_backend.py

```diff
@@ -271,6 +271,7 @@ def get_info(self, statistics: bool = False) -> dict:
             flag to request more detailed information about the content of the storage.
 
         :returns:
-            a nested dict with the relevant information.
+            a nested dict with the relevant information (with at least one key for `database`
+            and one for `repository`).
```

Comment:
Since this is the abstract class, it should not require that these keys are present. There could be some storage that doesn't have a separate database and repository

Reply:
Mmm ok, true, but as it is right now I need at least `database` to add the `summary` entry. Options for this:

1. We just accept this description for now and once we deal with `get_database_summary` we change it.
2. I put the output of `get_database_summary` in a separate`output_dict['database_summary']` instead of `output_dict['database']['summary']`.
3. I put the output of `get_database_summary` in `output_dict['database']`, risking overwriting what could come in that key from `storage.get_info` (which is currently empty).

Reply:
Bah, there is also the option of adding the `database` key if it is not there. I will default to that now and push, let me know if you prefer one of the other 3.